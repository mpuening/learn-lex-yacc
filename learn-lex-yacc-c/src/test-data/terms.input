a=1;
b=10;
c=10.01200;
d=(1);
e=(10);
f=(10.01200);