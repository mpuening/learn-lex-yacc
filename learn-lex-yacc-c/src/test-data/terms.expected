a = 1.000000
b = 10.000000
c = 10.012000
d = 1.000000
e = 10.000000
f = 10.012000
